### ISSUE TITLE
When media cannot be played, error message in media player should be keyboard accessible

### ISSUE THREAD

**Issue by elynema** [samvera-labs/ramp#660]
# Description

During crawls of Archivo, SiteImprove seems to find several media items where media cannot be played back. I'm not sure why this is happening, as the media plays fine when I visit the page, even across all different quality levels.

However, this does demonstrate that when there is an error message displaying in the Ramp media player, it should be accessible via tab/keyboard so that screenreaders can access it. 

<img width="1358" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2MzQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f8acd0a50ede8edc9c186bc80b3fd359473d7b33/image.png" alt="image.png" />

# Done Looks Like

- [ ] Test a media item that is not accessible to play because it has just been uploaded and a media item that is inaccessible in a playlist due to permissions and a media item that is inaccessible because it is a CDL item. 
- [ ] Verify that a screenreader and tab navigation can progress to read this error message
- [ ] If the skip to main link normally directs users to the play button on the media player, in this instance it should direct users to the error message.

**Comment by joncameron** [comment]
Could be resolved after 7.8 upgrade; will need to re-check after upgrade to see if this is resolved or is related to some other error.

**Comment by joncameron** [comment]
1 point for test post-upgrade; re-evaluate afterward.

**Comment by joncameron** [comment]
👍
